collectors/ps_constants/tests/cassettes/test_core/TestPsConstantsCollection.test_fetch_ps_constants.yaml

```diff
@@ -1358,14 +1344,13 @@ interactions:
       User-Agent:
       - python-requests/2.31.0
     method: GET
-    uri: https://gitlab.cee.redhat.com/prodsec-dev/ps-constants/-/raw/master/data/ubi_packages.yml?job=build
+    uri: https://example.com/prodsec-dev/ps-constants/-/raw/master/data/ubi_packages.yml?job=build
   response:
     body:
       string: "# This list of packages is used to indicate the UBI flag in the SFM2.
         \n# This list is NOT used to calculate SLA.\n\n\n# list of packages / components
         in UBI images\n# separate list provided for each RHEL version, keys need to
-        match ps module\n# names\n# https://projects.engineering.redhat.com/browse/PSIRT-730\n#
```

Comment:
Nitpick: I'd prefer the cassettes to retain their contents and have just the URLs replaced because the cassettes capture files that might already look different. I think it's ok to try to rerecord the cassettes and then check that bugs are filed for updating the relevant functionality, but I also think that doing it in this PR would duplicate work, for example work done in OSIDB-1876 related to changes in compliance-priority.yml.